### DISCUSSION TITLE
How can I support you?

### DISCUSSION THREAD

**Discussion by sachintha180** [iewnfod/CAIE_Code#89] [Q&A] [answered]
Hi, I've tried to support you via the "Sponsor" link on GitHub, but I'm always met with the following error:

"请填写验证码" - is there any other way we can support you?

Thanks for the amazing work! I'm a CIE Computer Science teacher, and this is really helpful.

**Comment by lightumcc** [comment, marked as answer]
Thank you for your interest in this project, and sorry for the inconvenience.
> Since we didn't expect to have international users sponsoring this project.

We've just added a [Patreon](https://patreon.com/createchstudio) link to the repo's home page, so hopefully that will help.

Thanks again for your support, enjoy your teaching journey and if you have any questions, please feel free to submit an issue!

**Reply by sachintha180** [reply]
Thank you, just supported!

One question, are single line declarations variable declarations supported in this version?

For example, rather than saying:
```
DECLARE i: INTEGER
DECLARE j: INTEGER
DECLARE k: INTEGER
```

can we do:
```
DECLARE i, j, k: INTEGER
```

Because single line declarations always failed to interpret on my machine. I can follow up with more specific errors if required.

**Reply by lightumcc** [reply]
Yes, this kind of declaration is supported by CPC.

From my perspective, this kind of method only appears in the marking scheme of past papers, but not in the PseudoCode Guide. If you have further confirmation, please let me know. Thanks.

Please try `cpc -u` if you cannot access this function.

```bash
CAIE Pseudocode Interpreter v0.1.4 (stable/f0fc00a)
Using [ PyPy 3.10.13 ] on Darwin
Repository at https://github.com/iewnfod/CAIE_Code/
Copyright © 2024 Iewnfod.
All Rights Reserved.
> DECLARE i, j, k: INTEGER
> OUTPUT i
0
```

> We are just students, we cannot confirm which method is correct, so we actually provide both kind of declaration methods.
